proto/grpc/federation/federation.proto

```diff
@@ -216,6 +218,16 @@ message RetryPolicyExponential {
   optional uint64 max_retries = 5;
 }
 
+// HeaderConfig define the response header metadata configuration.
+message HeaderConfig {
+  // condition to set the header. If true, the header is set. The condition is evaluated as CEL expression.
+  optional string if = 1;
+  // a kye for the metadata.
+  string key = 2;
+  // a value for the metadata. The value is evaluated as CEL expression.
+  repeated string value = 3;
```

Comment:
The same here 
https://github.com/mercari/grpc-federation/pull/90/files#r1461410777